docs/gambit/mutation-verifier.md

```diff
@@ -0,0 +1,99 @@
+# Using Gambit with the Prover
```

Comment:
It would be really good to set up https://github.com/Certora/ERC20Example with a gambit config and use it for examples.

Reply:
yes! Let me do that as a separate PR if that is ok. Like I was saying, we are waiting for the packaging to be done after which this will be much easier.